renku/ui/cli/mergetool.py

```diff
@@ -17,17 +17,24 @@
 # limitations under the License.
 """Custom git merge tool for renku metadata.
 
-Support merging Renku metadata
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Description
+~~~~~~~~~~~
 
 Renku stores all of its metadata in the repository, in compressed form.
-When working in multiple branches with Renku, this metadata needs to be merged when a git merge is made.
-To support users when doing this, Renku provides a custom merge tool that takes care of merging the metadata.
-
-The merge tool is set up automatically when creating a new project or when using ``renku clone`` to clone a Renku
-project.
+When working in multiple branches with Renku, this metadata needs to be merged
+when a git merge is made. To support users when doing this, Renku provides a
+custom merge tool that takes care of merging the metadata.
 
+The merge tool is set up automatically when creating a new project or when
+using ``renku clone`` to clone a Renku project.
```

Comment:
Wasn't there before either, but maybe we should write something along the lines of "Merge tool configuration cannot be shared through remote repositories and has to be set up on each clone separately" ? `renku clone` covers it but something to let the user know that if they do a normal clone, this will be missing.

Reply:
Yes, that makes sense. Otherwise, users might believe that `renku clone` keeps everything alligned